src/services/ui/src/pages/dashboard/Lists/spas/consts.tsx

```diff
@@ -43,9 +43,9 @@ export const TABLE_COLUMNS = (props?: { isCms?: boolean }): OsTableColumn[] => [
         : BLANK_VALUE,
   },
   {
-    field: props?.isCms ? "cmsStatus.keyword" : "stateStatus.keyword",
+    field: props?.isCms ? "stateStatus.keyword" : "stateStatus.keyword",
     label: "Status",
-    cell: (data) => (props?.isCms ? data.cmsStatus : data.stateStatus),
+    cell: (data) => (props?.isCms ? data.stateStatus : data.stateStatus),
```

Comment:
Lines 46 and 48 here are both saying "If the user is CMS, show them the stateStatus, else, show them the stateStatus". Why wouldn't we want to show them the CMS value if they're CMS?

Reply:
Based on what we talked about, I think the check should be more like 
```
(props.isCms && !props.user.[cms-roles].includes(UserRole.HELPDESK) ? cmsStatus : stateStatus
```
That way we still show CMS statuses to everyone at CMS except the helpdesk as the AC says